### DISCUSSION TITLE
Is a self balancing boost hub viable?

### DISCUSSION THREAD

**Discussion by lummox** [pybricks/support#628] [Q&A]
Hi! Do you guys think a self-balancing robot based on the boost hub (17101) could be viable? Does anyone have any experience oder working code?

**Comment by laurensvalk** [comment]
The boost Move Hub doesn’t have a built in gyroscope (only an accelerometer), so it wouldn’t work with just the hub.

But it should be possible with the color distance sensor, similar to this: https://www.philohome.com/nxtway/nxtway.htm
